### DISCUSSION TITLE
Testing language extension locally? No option in language selector.

### DISCUSSION THREAD

**Discussion by spaceunifyfifty** [zed-industries/zed#10834] [Feature Requests]
I submitted a PR to the kotlin language extension that was merged https://github.com/zed-extensions/zed-kotlin/commit/e175918c3de463040b025fa7059a33993bd0fdcf

I am now trying to test the updated extension locally before submitting it as PR to the main zed-industries/extensions repository. 
1. I uninstall the Kotlin extension
2. I clone https://github.com/zed-extensions/zed-kotlin
3. I press `Install Dev Extension` in zed settings, select `zed-kotlin` locally and confirm that it is built.

However, when I open `.kt` files, there is no color syntax highlighting at all. Restarting the language server, reloading the window, and rebuilding the extension do not fix this. Also, when I click `language selector` I no longer see Kotlin. I only see Kotlin as an option in the language selector when installing the official Kotlin extension that is online in the marketplace already. I have tested the local development checked out to the same commit as the official extension on Zed currently is at, and the problem persists. 

I assume I need to do an additional step of letting Zed know that my custom cloned dev extension is a valid language option, so I am able to select it. Where to go from here?

**Comment by osiewicz** [comment]
Hm, I can't install it as a dev extension either. Could it be due to the fact that it's using a legacy .json manifest?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
